### ISSUE TITLE
OK to use BOLT-LMM results for LDSC analyses?

### ISSUE THREAD

**Issue by jpbeauch** [sihrus/riskprefsgwas#17]
From Patrick:
BOLT-LMM essentially does OLS regression of the residuals on the target SNP. The residuals come from a prediction of the outcome using all SNPs on chromosomes other than the target SNPs. So the results should not be thought of as Ridge coefficients since they have no shrinkage. If there is no stratification, they are equal to standard OLS estimates in expectation, but with tighter SEs.
Using LDSC on BOLT-LMM results seems a little funny since the magnitude of the chi-squared stats are calibrated using the LDSC results, I think. I guess it depends on what you'd like to do with them.

From jpbeauch:
Good question re: BOLT-LMM results and LDSC.
The beta’s should be unbiased and good, but the SE’s may be deflated with LDSC intercept as Patrick mentions, but that deflating factor should cancel out in the num and denom for LDSC partition and LDSC correlation, so we should be ok for these.
For LDSC intercept, probably meaningless since if I remember well BOLT-LMM already calibrates this
In other words, I think we’re fine for LDSC partition and LDSC correlation, but I’m making a note to think more about this
